### DISCUSSION TITLE
Unable to connect with FTP like SFTP

### DISCUSSION THREAD

**Discussion by EngrAhsanAli** [drakkan/sftpgo#1298] [General]
Hi

Although its a very cool tool to manage FTP but I am facing a problem to simply connect with FTP.
I have 2.5.0 and installed in ubuntu 22
I can create and use SFTP normally, but not FTP due the following issue.

The authentication works fine but after Listing Directories, it gets fails with the following error.
_Connection reset. The connection attempt was rejected. The server may be down, or your network may not be properly configured._

I have tried with port 21 and 2122 which are opened and seems not an issue. Below are the sftpgo.json snippet for FTP.

 ```
"ftpd": {
    "bindings": [
      {
        "port": 21,
        "address": "",
        "apply_proxy_config": true,
        "tls_mode": 0,
        "certificate_file": "",
        "certificate_key_file": "",
        "min_tls_version": 12,
        "force_passive_ip": "",
        "passive_ip_overrides": [],
        "client_auth_type": 0,
        "tls_cipher_suites": [],
        "passive_connections_security": 0,
        "active_connections_security": 0,
        "debug": false
      }
    ],
    "banner": "",
    "banner_file": "",
    "active_transfers_port_non_20": true,
    "passive_port_range": {
      "start": 50000,
      "end": 50100
    },
    "disable_active_mode": false,
    "enable_site": false,
    "hash_support": 0,
    "combine_support": 0,
    "certificate_file": "",
    "certificate_key_file": "",
    "ca_certificates": [],
    "ca_revocation_lists": []
  },


```

I have tried several times in new machines and even Windows server too, but the issue persists.

**Comment by drakkan** [comment]
Hello,

thank you for using SFTPGo. FTP and SFTP are completly different protocols, FTP requires more network configurations than opening a single TCP port.

As an open-source initiative, our resources are primarily dedicated to new feature development, which limits our capacity for providing complimentary support. To receive support please consider a [GitHub sponsorship](https://github.com/sponsors/drakkan) or one of our available [support plans](https://sftpgo.com/#pricing).
